build.gradle

```diff
@@ -11,15 +23,52 @@ targetCompatibility = 1.8
 repositories {
     mavenCentral()
 }
+configurations {
+    implementation {
+        exclude group: 'org.slf4j', module: 'slf4j-log4j12'
+        exclude group: 'javax.servlet', module: 'servlet-api'
+        exclude group: 'org.mortbay.jetty', module: 'servlet-api'
```

Comment:
what dependency clash are we working around here? Can we add it to a comment possibly?